.travis.yml

```diff
@@ -25,9 +25,12 @@ script:
   # - npm run test
   - npm run build:prod
 
-after_success:
-  - npm run deploy
-  # - npm run codecov
+deploy:
+  skip_cleanup: true
+  provider: firebase
+  token:
+    project: $TRAVIS_BRANCH
```

Comment:
Project should be at the top level: https://docs.travis-ci.com/user/deployment/firebase/